charts/renga/values.yaml

```diff
@@ -26,6 +26,8 @@ global:
     postgresDatabase: gitlabhq_production
     postgresUser: gitlab
     postgresPassword: gitlab
+    clientSecret: open_secret
+    rootPassword: root_password
```

Comment:
This doesn't need to be global AFAIK.
can we move it to the `gitlab section`? Rename the exisitng `gitlab.password` value to `gitlab.rootPassword` if desired.

Reply:
Well I think we want to provide sensible defaults in `charts/gitlab/values.yaml` and potentially override them in the top-level `values.yaml`. But you're right, these should not in any case be in a part of `global`.

Reply:
actually I think it does need to be global because both keycloak and gitlab need to access the value?

Reply:
No, that values is only found in `charts/gitlab/templates/secret.yaml`

Reply:
ah I thought you were referring also to `clientSecret` -- yes, `rootPassword` can be removed, I just did in 97319b0